slacktee.sh

```diff
@@ -193,22 +192,41 @@ function send_message()
 
 		username=$(escape_string "$username")
 
-		json="{\
-                  \"channel\": \"$channel\", \
-                  \"username\": \"$username\", \
-                  $message_attr \"icon_emoji\": \"$icon_emoji\", \
-                  \"icon_url\": \"$icon_url\" $parseMode}"
-		post_result=$(curl -X POST --data-urlencode \
-                  "payload=$json" "$webhook_url" 2> /dev/null)
-                if [[ $post_result != "ok" ]]; then
-                	err_exit 1 "$post_result"
-                fi
+		if [[ $mode == "streaming" ]]; then
+			if [[ -z "$streaming_ts" ]]; then
+				post_result=$(curl -d "token=$token&username=$username&icon_url=$icon_url&icon_emoji=$icon_emoji&$parseModeUrlEncoded&channel=$channel&text=$wrapped_message" -X POST https://slack.com/api/chat.postMessage 2> /dev/null)
+				if [ $? != 0 ]; then
+			    err_exit 1 "$post_result"
+				fi
+
+				# chat.update requires the channel id, not the name
+				streaming_channel_id=$(echo "$post_result" | python -c "import sys, json; print json.load(sys.stdin)['channel']")
```

Comment:
Line 203 and 206 adds a dependency to python. This is a significant change for slacktee users.
I know python is installed in the most of environment, but some users are using slacktee in the very lean environment and I can imaging this additional dependency makes them confused.

One of the selling point of slacktee is the minimum dependency and I cannot accept this change. If you need to extract the data from the response, please use `awk` command like line 707. I know it's not fun, but keeping the backward compatibility is really important for slacktee users. 

I appreciate your understanding.

Reply:
done. I'm just glad you wrote the awk, and I just needed to copy and tweak.